scripts/irods/test/test_iticket.py

```diff
@@ -820,6 +820,13 @@ def test_ticket_create_with_relative_paths_admin__issue_7103(self):
             self.admin.assert_icommand(['irm', '-rf', collection_1])
             self.admin.assert_icommand(['irm', '-rf', collection_2])
             self.admin.assert_icommand(['irm', '-f', home_data_object])
+            
+            self.admin.assert_icommand(['iticket', 'delete', collection_1_ticket_name])
```

Comment:
Might want to change `assert_icommand` to `run_icommand` just in case one fails.